SI/Sem.05/Pract.05/Task02.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+using namespace std;
+void swap(int& a, int& b)
+{
+	int temp = a;
+	a = b;
+	b = temp;
+}
+void SortEvenNumsAscending(int* arr, int Size)
```

Comment:
Be consistent in naming you functions. (Currently swap is camelCase, and the other is PascalCase)